src/main/java/net/sf/marineapi/ais/util/SpeedOverGround.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SpeedOverGround.java
+ * Copyright (C) 2018 Jyrki Jakobsson
+ *
+ * This file is part of Java Marine API.
+ * <http://ktuukkan.github.io/marine-api/>
+ *
+ * Java Marine API is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * Java Marine API is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Java Marine API. If not, see <http://www.gnu.org/licenses/>.
+ */
+package net.sf.marineapi.ais.util;
+
+import java.text.DecimalFormat;
+
+/**
+ * Checks a speed-over-ground value for validity.
+ *
+ * @author Jyrki Jakobsson
+ */
+public class SpeedOverGround {
```

Comment:
Maybe we should add separate tests for utils like this, too. Not a big isssue regarding this PR but currently the utils are only tested through messages, thus leaving some parts of code or edge values uncovered.

Reply:
I added a few tests for this and the Latitude & Longitude classes. Pretty trivial, but at least they document the difference between isCorrect and isAvailable.